pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -185,14 +185,16 @@ edk2.mkDerivation projectDscPath (finalAttrs: {
   '';
 
   postBuild =
-    lib.optionalString (stdenv.hostPlatform.isAarch || stdenv.hostPlatform.isLoongArch64) ''
-      (
-      cd ${buildPrefix}/FV
-      cp QEMU_EFI.fd ${fwPrefix}_CODE.fd
-      cp QEMU_VARS.fd ${fwPrefix}_VARS.fd
-      )
-    ''
-    + lib.optionalString stdenv.hostPlatform.isAarch ''
+    lib.optionalString
+      ((stdenv.hostPlatform.isAarch || stdenv.hostPlatform.isLoongArch64) && (fwPrefix != "CLOUDHV"))
+      ''
+        (
+        cd ${buildPrefix}/FV
+        cp QEMU_EFI.fd ${fwPrefix}_CODE.fd
+        cp QEMU_VARS.fd ${fwPrefix}_VARS.fd
+        )
+      ''
+    + lib.optionalString (stdenv.hostPlatform.isAarch && (fwPrefix != "CLOUDHV")) ''
```

Comment:
These conditionals are starting to look like a bit of a lack of separation of concerns.  Maybe at this point it would make more sense to just decouple OVMF and OVMF-cloud-hypervisor?